src/core/public/notifications/toasts/toasts_api.test.ts

```diff
@@ -146,6 +146,20 @@ describe('#remove()', () => {
   });
 });
 
+describe('#addInfo()', () => {
+  it('adds a info toast', async () => {
+    const toasts = new ToastsApi(toastDeps());
+    expect(toasts.addInfo({})).toHaveProperty('color', 'primary');
+  });
```

Comment:
Can we add a test asserting that the new `toastLifeTimeMs` option is properly propagated?

Reply:
@pgayvallet done